src/main/java/org/cancogenvirusseq/muse/service/SongScoreService.java

```diff
@@ -57,6 +57,7 @@ private Disposable createSubmitUploadDisposable() {
   }
 
   public Mono<Upload> submitAndUploadToSongScore(UploadEvent uploadEvent) {
+    log.info("Payload: {}", uploadEvent.getPayload());
```

Comment:
No problem leaving the extra logging if you think this could be useful for troubleshooting in prod, but lets maybe reduce it to debug level and add some more description like `UploadEvent payload for song:`

Reply:
Reduced logging level to debug. Changed text.